roles/install-k8s/tasks/main.yml

```diff
@@ -41,6 +41,7 @@
 - name: Enable and start kubelet
   systemd:
     name: kubelet
+    daemon_reexec: true
```

Comment:
The kubelet service is not natively detected unless a daemon_reexec is done. 
```
TASK [install-k8s : Enable and start kubelet] ************************************
fatal: [135.90.79.46]: FAILED! => {"changed": false, "msg": "Could not find the requested service kubelet: host"}
fatal: [135.90.79.45]: FAILED! => {"changed": false, "msg": "Could not find the requested service kubelet: host"}
```

Reply:
but I don't see this error in any of the latest prow runs, e.g: https://storage.googleapis.com/ppc64le-kubernetes/logs/periodic-kubernetes-containerd-conformance-test-ppc64le/1638119113425424384/build-log.txt

Reply:
With the changes made for CRI-O, the additional overhead of reloading the systemd is now required to detect the kubelet service. This was not observed in the case of Containerd as runtime and requires some analysis.

 Hence, this quick-fix was added and tested with the Terraform + Ansible flow for both runtimes.

Reply:
okay, later create a task to see if there is any solution exist to avoid this.